### PR TITLE
Make external links in Spanish version point to Spanish-language URLs

### LINKED ISSUE #199: Spanish links should direct to Spanish EDD pages
Right now, if a Spanish user clicks on a link in the guide, it will direct them to the English version of the expected page. We should update the links to go to the Spanish version. 

https://docs.google.com/document/d/160ZmesUtMzUCTIk-nXEFANQnjqt-WBa3GbuCfkybVWg/edit

### PR DESCRIPTION
**Pull request by kalvinwang**
This PR makes external links in Spanish version point to Spanish-language URLs where available, and matches the content doc. 

I reviewed by manually going through every link in the Spanish version of the guide.

===

Resolves #199

- [x] Snapshots updated, if output HTML/JS has changed (`npm run test:update-snapshots`)
- [ ] Changes reviewed on mobile using devtools, if output HTML/CSS has changed
- [x] Spanish checked by adding `?lng=es` to URL e.g. `/guide/benefits?lng=es`, if Spanish updated
- [ ] Automated tests added, if new functionality added
- [ ] Documentation (e.g. READMEs) updated, if relevant
- [ ] IE11 and Edge compatibility confirmed via manual testing in Browserstack, if new libraries added or newish JS/HTML/CSS features used for the first time in this codebase
- [ ] Accessibility & performance audit run using Google Lighthouse, if major changes made

### CODE REVIEW

File: public/locales/es/translation.json [hunk 1]
```diff
@@ -42,6 +42,19 @@
     "siteMap": "Site Map",
     "copyright": "Copyright © 2020 State of California"
   },
+  "links": {
+    "edd-bpo": "https://www.edd.ca.gov/Benefit_Programs_Online_Espanol.htm",
+    "edd-di-claim": "https://edd.ca.gov/Disability/How_to_File_a_DI_Claim_in_SDI_Online.htm",
```
Comment (kalvinwang) [inline public/locales/es/translation.json:47]: this matches the doc and there doesn't appear to be a spanish version of this page

File: src/client/components/TabPaneContent3/index.js [hunk 2]
```diff
@@ -19,10 +19,7 @@ function TabPaneContent3() {
             </strong>{" "}
             We'll let you know what your WBA is, based on our wage records. For
             an initial estimate of your WBA, use our{" "}
-            <a href="https://www.edd.ca.gov/Unemployment/UI-Calculator.htm">
-              UI benefit calculator
-            </a>
-            .
+            <a href={t("links.edd-filing-claim")}>UI benefit calculator</a>.
```
Comment (sharonwarner) [inline src/client/components/TabPaneContent3/index.js:22]: This should be the UI calculator.

File: src/client/components/TabPaneContent3/index.js [hunk 3]
```diff
@@ -81,9 +76,8 @@ function TabPaneContent3() {
         <Trans t={t} i18nKey="tab3-p7">
           We understand that it’s a difficult time to look for a job right now.
           You are <strong>currently not</strong> required to upload your resume
-          to{" "}
-          <a href="https://www.caljobs.ca.gov/vosnet/Default.aspx">CalJOBS</a>,
-          during the COVID-19 pandemic.
+          to <a href={t("links.caljobs")}>CalJOBS</a>, during the COVID-19
+          pandemic.
```
Comment (sharonwarner) [inline src/client/components/TabPaneContent3/index.js:80]: This isn't showing up correctly in the snapshot for some reason.

File: src/client/components/TabPaneContent5/index.js [hunk 4]
```diff
@@ -41,11 +41,7 @@ function TabPaneContent5() {
           history. It is also available to people who have exhausted their
           regular UI claims, are serving penalty weeks on their claim, or whose
           claim may be disqualified. PUA supports claims between February 2 and
-          December, 2020.{" "}
-          <a href="https://edd.ca.gov/about_edd/coronavirus-2019/pandemic-unemployment-assistance.htm">
-            Read more about PUA
-          </a>
-          .
+          December, 2020. <a href={t("links.edd-pua")}>Read more about PUA</a>.
```
Comment (sharonwarner) [inline src/client/components/TabPaneContent5/index.js:44]: This is also not showing up correctly in the snapshot.

File: src/client/components/TabPaneContent3/index.js (hunk 3 above)
Comment (sharonwarner) [reply src/client/components/TabPaneContent3/index.js:80]: Yeah, it looks like it's not showing up in the UI either.

Comment (kalvinwang) [reply src/client/components/TabPaneContent3/index.js:80]: Huh. Somehow the node numbering changed from 4 to 3. I guess the URL string counted as a text node and the t() does not count as a node at all? i18next is such a mystery...

Thanks!

File: src/client/components/TabPaneContent3/__snapshots__/index.test.js.snap [hunk 5]
```diff
@@ -24,7 +24,7 @@ exports[`<TabPaneContent3 /> renders the TabPaneContent component 1`] = `
           </strong>
            We'll let you know what your WBA is, based on our wage records. For an initial estimate of your WBA, use our 
           <a
-            href="https://www.edd.ca.gov/Unemployment/UI-Calculator.htm"
+            href="https://www.edd.ca.gov/Unemployment/ui-calculator.htm"
```
Comment (kalvinwang) [inline src/client/components/TabPaneContent3/__snapshots__/index.test.js.snap:27]: these both resolve to the same page

**Comment by kalvinwang** [comment]
Thanks for catching those! Fixed

**Pull request merged**